www/content/tutorials/preview-mode/create-oauth-client.mdx

```diff
@@ -27,13 +27,17 @@ As from `next-drupal 1.5`, user roles are used for OAuth scopes. The scopes are
 
 Next, assign the following permissions to the newly created role.
 
-- Bypass content access control
-- Issue subrequests
-- View user information
+- 'access content'
+- 'access user profiles'
+- 'issue subrequests'
+- 'view all revisions'
+- 'view any unpublished content'
+- 'view latest version'
+- 'view media'
```

Comment:
I tried out these new permissions on a Drupal 10.1 site and ran into a snag. 
 
What version of Drupal did you use to create this list?

In Drupal 10.1 some of those permissions don't exist. Best I can tell the permissions are:
- 'View all media revisions'
- 'View media'
- 'View own unpublished media'
- 'View all revisions'
- 'View published content'
- 'View own unpublished content'
- 'Issue subrequests'
- 'View user information'

What heading is "view latest version" under? I don't see anything in 10.1 that matches that.

Reply:
These were from a Drupal 9.5.11 site.

On our site, `view latest version` is underneath the `Content Moderation` heading, and was required for viewing forward revisions (i.e. new draft revision of previously published content) in our setup

![Screenshot 2023-11-29 at 10 14 27 AM](https://github.com/chapter-three/next-drupal/assets/11279744/b90862e0-f746-4649-87d8-3bd74a786a22)

Reply:
> On our site, view latest version is underneath the Content Moderation heading

Aha! I didn't have that installed. That module is also required for the "View any unpublished content" permission. Without that module, the only perms that mention "unpublished" are the ones under the "Node"/"Media" modules for "own unpublished".

And, yes, having the "View any unpublished content" permission is required to allow the Next.js User to view unpublished content owned by regular content editors.

I think we should add the module heading for each of these permission so that users (like me!) don't get confused about which permissions to use. Adding the module heading will also give a hint on how to get that permission if it isn't listed on a user's website.